src/assets/app.scss

```diff
@@ -196,17 +207,27 @@ body {
       }
     }
   }
-
+  .media-left {
+    pointer-events: none;
+    z-index: 1;
+  }
   .media-content {
     overflow: hidden;
     text-overflow: inherit;
   }
+  .infolink {
+    font-family: "Font Awesome 5 Free";
```

Comment:
Have you tried without this line? as we use the `<i>` tag with the css class, I don't think that's needed (other icons works without it)

Reply:
I'll take a look right now.

Reply:
Specifically line 219 or the entire block?

Reply:
Just the line 219 yes.

Reply:
Any update on this PR?

Reply:
Any update?